ingestion/src/metadata/utils/credentials.py

```diff
@@ -54,7 +54,8 @@ def set_google_credentials(gcs_credentials: GCSCredentials) -> None:
     if isinstance(gcs_credentials.gcsConfig, GCSCredentialsPath):
         os.environ[GOOGLE_CREDENTIALS] = str(gcs_credentials.gcsConfig.__root__)
         return
-
+    if gcs_credentials.gcsConfig.projectId is None:
```

Comment:
Could you please add an INFO log here, something like `no credentials sent, using the current environment permissions`